src/token_request_handler.ts

```diff
@@ -46,6 +51,24 @@ export class BaseTokenRequestHandler implements TokenRequestHandler {
     return (response as TokenErrorJson).error === undefined;
   }
 
+  performRevokeTokenRequest(
+      configuration: AuthorizationServiceConfiguration,
+      request: RevokeTokenRequest): Promise<boolean> {
+    let revokeTokenResponse = this.requestor.xhr<any>({
+      url: configuration.revocationEndpoint + '?' + this.utils.stringify(request.toStringMap()),
+      dataType: 'json',  // adding implicit dataType
+      method: 'GET'
```

Comment:
shouldn't this be a POST and the token be sent in the body ? - https://tools.ietf.org/html/rfc7009#section-2.1